incense/experiment.py

```diff
@@ -166,12 +169,11 @@ def _load_metrics(metrics_file: Path) -> Dict[str, pd.Series]:
     @staticmethod
     def _load_artifacts(run_dir: Path) -> Dict[str, Artifact]:
         artifacts = {}
-        reserved = {"run.json", "cout.txt", "metrics.json", "config.json"}
+        reserved = {"run.json", "cout.txt", "metrics.json", "config.json", "info.json"}
         artifact_paths = (p for p in run_dir.iterdir() if p.name not in reserved and not p.is_dir())
         for artifact_path in artifact_paths:
-            artifact_file = artifact_path.open(mode="rb")
             name = artifact_path.name
-            artifacts[name] = Artifact(name, artifact_file)
+            artifacts[name] = Artifact(name, artifact_path)
```

Comment:
Actually surprised that this works. However, the Artifact type expects something file like to keep it compatible with the MongoDB variant. For a change like this we would have to define a general protocol to abstract over local and MongoDB files.

Reply:
That is a good point. The main reason for this change was that the previous version opened artifact files without closing them, which lead to errors when loading a large number of experiments at once.